content/ar/docs/setup/best-practices/certificates.md

```diff
@@ -0,0 +1,281 @@
+---
+title: شهادات البنية التحتية للمفتاح العام PKI و المتطلبات
+reviewers:
+- sig-docs-ar-reviews
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+
+كوبرنيتيز يتطلب شهادات البنية التحتية للمفتاح العام PKI للمصادقة عبر نظام أمان طبقة النقل (Transport Layer Security - TLS)
+إذا قمت بتثبيت كوبرنيتيز باستخدام [kubeadm](/docs/reference/setup-tools/kubeadm/), الشهادات
```

Comment:
check correct spelling for Kubernetes in arabic here : 
https://deploy-preview-46171--kubernetes-io-ar-staging.netlify.app/ar/docs/contribute/localization/
and here https://github.com/kubernetes/website/issues/45078